doc/can-query-logic.md

```diff
@@ -330,45 +366,82 @@ For example, to change queries to use `where` instead of `filter` so that querie
 made like:
 
 ```js
-Todo.getList({
-    where: {complete: true}
-})
+import {DefineMap, DefineList, realtimeRestModel, QueryLogic} from "can";
+import {Todo, todoFixture} from "//unpkg.com/can-demo-models@5";
+
+// creates a mock todo api
+todoFixture(5);
+
+Todo.list = DefineList.extend("TodoList", {
+  "#": {Type: Todo}
+});
+
+const todoQueryLogic = new QueryLogic(Todo);
+
+Todo.connection = realtimeRestModel({
+  url: "/api/todos/{id}",
+  Map: Todo,
+});
+
+Todo.getList({filter: {complete: true}}).then(todos => {
+  todos.forEach(todo => {
+    console.log(todo.name); // logs completed todos
+  });
+});
+
 ```
+@codepen
+@highlight 18,22,only
 
-You can use the `options`'s `toQuery` and `toParams` functions
+You can use the `options`' `toQuery` and `toParams` functions
 to set the `filter` property value to the passed in `where` property value.
 
 ```js
-// DEFINE YOUR TYPE
-const Todo = DefineMap.extend({...});
+import {DefineMap, QueryLogic, realtimeRestModel} from "can";
+import {Todo, todoFixture} from "//unpkg.com/can-demo-models@5";
+
+todoFixture(5);
 
 // CREATE YOUR QUERY LOGIC
-var todoQueryLogic = new QueryLogic(Todo, {
-    // Takes what your service expects: {where: {...}}
-    // Returns what QueryLogic expects: {filter: {...}}
-    toQuery(params){
-        var where = params.where;
-        delete params.where;
-        params.filter = where;
-        return params;
-    },
-    // Takes what QueryLogic expects: {filter: {...}}
-    // Returns what your service expects: {where: {...}}
-    toParams(query){
-        var where = query.filter;
-        delete query.filter;
-        query.where = where;
-        return query;
-    }
+const todoQueryLogic = new QueryLogic(Todo, {
+  // Takes what your service expects: {where: {...}}
+  // Returns what QueryLogic expects: {filter: {...}}
+  toQuery(params){
+    const where = params.where;
+    delete params.where;
+    params.filter = where;
+    return params;
+  },
+  // Takes what QueryLogic expects: {filter: {...}}
+  // Returns what your service expects: {where: {...}}
+  toParams(query){
+    const where = query.filter;
+    delete query.filter;
+    query.where = where;
+    return query;
+  }
+});
+
+Todo.list = DefineList.extend("TodoList", {
+  "#": {Type: Todo}
 });
 
 // PASS YOUR QueryLogic TO YOUR CONNECTION
-realTimeRest({
-  url: "/todos",
+Todo.connection = realtimeRestModel({
+  url: "/api/todos/{id}",
   Map: Todo,
   queryLogic: todoQueryLogic
 });
+
+Todo.getList({filter: {complete:true}}).toQuery().then(todos => {
```

Comment:
`toQuery()`?  I think this is broken

Reply:
Commented out codepen for now as I'm not sure this is suppose to work.

Reply:
promises don't have a `.toQuery()` method.